draft-ietf-quic-transport.md

```diff
@@ -2489,7 +2483,8 @@ least one of them is processable by the client.  Similarly, a peer might be
 unable to read 1-RTT packets, so an endpoint SHOULD send CONNECTION_CLOSE in
 Handshake and 1-RTT packets prior to confirming the handshake; see Section 4.1.2
 of {{QUIC-TLS}}.  These packets can be coalesced into a single UDP datagram; see
-{{packet-coalesce}}.
+{{packet-coalesce}}.  After the handshake is confirmed, an endpoint MUST send
```

Comment:
```suggestion
{{packet-coalesce}}.

After the handshake is confirmed, an endpoint MUST send
```

Reply:
I would also suggest moving this up to the earlier para after the sentence "Generally, this means sending the frame in a packet with the highest level of packet protection to avoid the packet being discarded.". Then, change the beginning of the next sentence to "However, prior to handshake confirmation ...".

Reply:
Yep, that makes more sense.  Thanks.